syntax = "proto3";
package calendar_events;





/*
CreateEvent请求
*/
message CreateEventRequest {
    /*
    事件名称
    */
    string eventName = 1 ;
    /*
    事件相关成员
    */
    repeated string members = 2 ;
    /*
    事件类型
    */
    string type = 3 ;
    /*
    开始日期 2020-02-02
    */
    string startDay = 4 ;
    /*
    开始时间 09:30:00
    */
    string startTime = 5 ;
    /*
    结束日期 2020-02-02
    */
    string endDay = 6 ;
    /*
    结束时间 09:30:00
    */
    string endTime = 7 ;
    /*
    事件备注
    */
    string memo = 8 ;
    /*
    事件的等级
    */
    string level = 9 ;
}

/*
CreateEvent返回
*/
message CreateEventResponse {
    /*
    事件id
    */
    string id = 1 ;
}

/*
CreateEventApi返回
*/
message CreateEventResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    calendar_events.CreateEventResponse data = 4 ;
}

